.eslintrc.cjs

```diff
@@ -27,5 +28,6 @@ module.exports = {
 				parser: '@typescript-eslint/parser'
 			}
 		}
-	]
+	],
+	rules: { 'svelte/no-at-html-tags': 'off' }
```

Comment:
This is required so I can render markdown. Normally, this rule is to protect against cross-site scripting attacks, but the markdown in this case comes from Kiara's in-code documentation only, so I know it's safe